packages/common/src/eips/1153.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "EIP-1153",
+  "number": 1153,
+  "comment": "Transient Storage",
+  "url": "https://eips.ethereum.org/EIPS/eip-1153",
+  "status": "Review",
+  "minimumHardfork": "chainstart",
+  "requiredEIPs": [],
+  "gasConfig": {},
+  "gasPrices": {
+    "tstore": {
+      "v": 200,
+      "d": "Base fee of the TSTORE opcode"
+    },
+    "tload": {
+      "v": 200,
```

Comment:
Same here, should be 100 gas per EIP spec? (unless I'm misreading)

Reply:
You are correct, I need to update the values. There still isn't complete consensus on what they should be. They cannot be so cheap that a tx that uses a full block's worth of gas to store as many key/value pairs as possible can oom a node on consumer hardware